common/v2/components/AmountInput.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import { Input } from '@mycrypto/ui';
+import { Asset } from 'v2/types';
+import { monospace } from 'v2/theme';
+import { TSymbol } from 'v2/types/symbols';
+import AssetIcon from './AssetIcon';
+import Typography from './Typography';
+interface Props {
+  asset: Asset;
+  value: string;
+  placeholder?: string;
+  onChange(event: any): void;
+  onBlur?(event: any): void;
+}
+
+const SInput = styled(Input)`
+  font-family: ${monospace};
```

Comment:
Using monospace for numbers is strange.
The designs don't show it: https://www.figma.com/file/BY0SWc75teEUZzws8JdgLMpy/%5BMyCrypto%5D-GAU-Master?node-id=0%3A25003

I added a comment on https://app.clubhouse.io/mycrypto/story/3527/send-assets-improvements-amount-input-and-monospace to request more info.

I'm happy to merge without this.